javav2/example_code/autoscale/src/test/java/AutoScaleTest.java

```diff
@@ -34,68 +42,67 @@ public class AutoScaleTest {
     private static String instanceId2="";
     private static String launchTemplateName="";
-    private static String serviceLinkedRoleARN="";
     private static String vpcZoneId="";
 
     @BeforeAll
     public static void setUp() throws IOException {
-
         autoScalingClient = AutoScalingClient.builder()
-                .region(Region.US_EAST_1)
-                .credentialsProvider(ProfileCredentialsProvider.create())
-                .build();
-
+            .region(Region.US_EAST_1)
+            .credentialsProvider(EnvironmentVariableCredentialsProvider.create())
+            .build();
+
+        Random random = new Random();
+        int randomNum = random.nextInt((10000 - 1) + 1) + 1;
+
+        // Get the values to run these tests from AWS Secrets Manager.
+        Gson gson = new Gson();
+        TestValues myValues = gson.fromJson(String.valueOf(getSecretValues()), TestValues.class);
+        groupName = myValues.getGroupName()+randomNum;
+        launchTemplateName = myValues.getLaunchTemplateName();
+        vpcZoneId = myValues.getVpcZoneId();
+        groupNameSc = myValues.getGroupNameSc()+randomNum;
+
+       // Uncomment this code block if you prefer using a config.properties file to retrieve AWS values required for these tests.
+       /*
         try (InputStream input = AutoScaleTest.class.getClassLoader().getResourceAsStream("config.properties")) {
-
             Properties prop = new Properties();
             if (input == null) {
                 System.out.println("Sorry, unable to find config.properties");
                 return;
             }
 
             prop.load(input);
-            groupName = prop.getProperty("groupName");
+            groupName = prop.getProperty("groupName")+randomNum;
             launchTemplateName = prop.getProperty("launchTemplateName");
-            serviceLinkedRoleARN = prop.getProperty("serviceLinkedRoleARN");
             vpcZoneId = prop.getProperty("vpcZoneId");
-            groupNameSc = prop.getProperty("groupNameSc");
+            groupNameSc = prop.getProperty("groupNameSc")+randomNum;
 
         } catch (IOException ex) {
             ex.printStackTrace();
         }
-    }
-    @Test
-    @Order(1)
-    public void checkValues() {
-        assertNotNull(autoScalingClient);
-        assertNotNull(groupName);
-        assertNotNull(instanceId);
-        assertNotNull(launchTemplateName);
-        assertNotNull(serviceLinkedRoleARN);
-        assertNotNull(vpcZoneId);
-        System.out.println("Test 1 passed");
+        */
     }
 
     @Test
-    @Order(2)
+    @Order(1)
     public void createAutoScalingGroup() {
-        assertDoesNotThrow(() -> CreateAutoScalingGroup.createAutoScalingGroup(autoScalingClient, groupName, launchTemplateName, serviceLinkedRoleARN, vpcZoneId));
+        assertDoesNotThrow(() -> CreateAutoScalingGroup.createAutoScalingGroup(autoScalingClient, groupName, launchTemplateName, vpcZoneId));
         System.out.println("Test 2 passed");

```

Comment:
Isn't this test 1 now? Also, are these comments needed? The test runner should be reporting this information.